### DISCUSSION TITLE
Someone is requesting money otherwise he is gonna publish my private repo, how is that possible?

### DISCUSSION THREAD

**Discussion by f1af1y** [community/community#37813] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,

I had a weird and non understandable situation lately with an anonymous person sending me a screenshot of a private repo and trying to get money or he sending it public.

Wanted to make sure that he has no access to my GitHub so I verified a few steps:
1/ Verified if any collaborators were added (they were none) and that the repo was private (yes it was private)
2/ Changed my password
3/ Double Auth was already enabled
4/ Delete all app access that I don't use (now only 1 app has access, Netlify)
5/ Revoke all SSH and GPG keys (They were none)

At that point I thought he might have getting the code from somewhere else, maybe Netlify as the git repo was deployed there. So I did that in Netify:
1/ Change my password, profile was create with the GitHub account so I created a password.
2/ Added double Auth
3/ Made sure no personal access tokens were created
4/ Verified no collaborator was added to the team

Am I missing something or is it possible for anyone to find the code directly from the website url or something else ?

Thank you for your help

**Comment by airtower-luna** [comment, marked as answer]
That looks about right as far as your account is concerned. You might want to check the security of your own system, too, to make sure an attacker can't get data directly from you, or mess with your connections to GitHub.

Consider getting law enforcement involved: Whether or not the screenshot is real and whether or not they really had access, extortion is a pretty serious crime.

**Reply by f1af1y** [reply]
Thank you for your answer!
I'll definitely delete and reinstall my system to make sure everything is secured again.

Well, that's something I might consider - not first thing it came up to my mind but might be the right way to deal with it.

**Comment by chasdse** [comment]
This situation, known as "ransomware," is illegal and punishable by [law](https://whatisnotlegal.com/). It is a form of extortion and is considered a serious crime. It is not acceptable to threaten to publish someone's private information or files without their consent. If you are being threatened with this type of behavior, you should contact the authorities immediately. Additionally, it is recommended to ensure regular backups of your important files, and to be aware of phishing or suspicious emails or messages.

**Comment by Alessandro-Battiato** [comment]
Isn't there a way to prevent these things automatically with GitHub in some sort of way? Like some security measures, as to make something like this literally impossible from happening
